tests/other_test.go

```diff
@@ -63,3 +74,65 @@ func TestRunOverrideExample(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunVsiExtention(t *testing.T) {
```

Comment:
@Aashiq-J why was this moved to other_test.go ? Won't it mean we have a test gap in PRs where this DA is not getting tested now?